netbout-client/src/main/java/com/netbout/client/RtAlias.java

```diff
@@ -108,11 +110,24 @@ public void photo(final URI uri) {
 
     @Override
     public String email() throws IOException {
-        return this.request.fetch()
-            .as(XmlResponse.class)
-            .xml()
-            .xpath("/page/alias/email/text()")
-            .get(0);
+        final XML body = this.request.fetch().as(XmlResponse.class).xml();
+        final List<String> lverified = body
```

Comment:
@CarlosGines Let's get rid of `lverified` variable and inline `body.xpath()` in the condition